src/checkers/inference/solver/backend/AbstractFormatTranslator.java

```diff
@@ -24,15 +24,18 @@
 import checkers.inference.util.ConstraintVerifier;
 
 /**
- * Created by mier on 08/11/17.
+ * Abstract implementation of FormatTranslator that all concrete FormatTranslator should extend from.
+ * It has the necessary lattice that will be used and already setup the abstract structure and provided
+ * default implementations so that you only need to override minimum methods your FormatTranslator is
```

Comment:
Avoid using personal pronouns (you, we, us..) in code documents as at the most of times they are ambiguous.

Substitute personal pronouns with the clearly defined entities, e.g.

instead of:

"so that you only need to override minimum methods your FormatTranslator is interested."

refactor it to something like:

"so that sub-class of this abstract formatTranslator only need to override minimum methods that it is interested."